index.html

```diff
@@ -72,6 +72,27 @@
           <a>standardized payment method identifier</a>.
         </li>
       </ul>
+      <section>
+        <h3>
+          Validity
+        </h3>
+        <p>
+          The steps to <dfn>validate a payment method identifier</dfn> with a
+          <var>pmi</var> string are given by the following algorithm. It
```

Comment:
Nit: both here and in other algorithms you seem to put the type after the variable name, but that's pretty unusual in web specs. "string _pmi_" and "URL _url_" would be more expected.